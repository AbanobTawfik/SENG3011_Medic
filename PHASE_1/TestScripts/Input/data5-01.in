#!/bin/sh
# max is 1

start_date=2015-01-01T00:00:00
end_date=2015-01-31T23:59:59
max=1
