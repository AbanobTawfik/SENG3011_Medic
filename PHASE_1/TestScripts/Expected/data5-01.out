{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "articles": [
        {
            "url": "url23",
            "reports": [
                {
                    "diseases": [],
                    "syndromes": [],
                    "event_date": "2015-01-01 xx:xx:xx to 2015-01-31 xx:xx:xx",
                    "locations": []
                }
            ],
            "headline": "Headline of url23",
            "main_text": "This is the main text for url23.",
            "date_of_publication": "2015-12-31 12:00:00"
        }
    ]
}
